lib_bgp_data/blacklist/tests/test_blacklist_parser.py

```diff
@@ -67,9 +67,9 @@ def test_run(self):
         """Test that should make sure that there are no errors and that
         the table is populated with the correct amount of ASNs"""
         # Run the parser
-        self.parser.run()
-        # Get the raw data. Done right after running parser for
-        # better precision.
+        parser = Blacklist_Parser()
+        parser.run()
```

Comment:
the run function will hide any errors. You should use _run. Also, did I request that the parser be moved into every file? I think I meant just that it should be initialized at the start of every test